src/Juvix/Compiler/Core/Extra/Value.hs

```diff
@@ -0,0 +1,36 @@
+module Juvix.Compiler.Core.Extra.Value where
+
+import Juvix.Compiler.Core.Data.InfoTable
+import Juvix.Compiler.Core.Extra.Base
+import Juvix.Compiler.Core.Info qualified as Info
+import Juvix.Compiler.Core.Info.ExpansionInfo (kExpansionInfo)
+import Juvix.Compiler.Core.Language
+import Juvix.Compiler.Core.Language.Value
+
+toValue :: InfoTable -> Node -> Value
+toValue tab = \case
+  NCst Constant {..} -> ValueConstant _constantValue
+  NCtr Constr {..} ->
```

Comment:
I'd prefer to have `goConstr`, `goLam` in the where block to improve readability

Reply:
done